README.md

```diff
@@ -37,6 +37,28 @@ config :shoehorn,
   app: Mix.Project.config()[:app]
 ```
 
+## SSH Console Port
+
+By default, `nerves_pack` will start an IEx console on port 22, this can be overriden
+by specifying `:ssh_console_port` in the config. The SFTP subsystem is also enabled
+so that you can transfer files back and forth as well. To disable this feature,
+set `:ssh_console_port` to `nil`.  This console will use the same ssh public
+keys as those configured for `:nerves_firmware_ssh`. Usernames are ignored.
+
+```elixir
+config :nerves_pack, ssh_console_port: 2222
```

Comment:
I might favor `ssh_port` since its more commonly referred to as `ssh` rather than `ssh_console`

Reply:
@jjcarstens Do you prefer `sshd_port` or `ssh_port`? The first option seems to be the most correct but I am also the type of person who would spend way too much time finding a type if I forgot the `d`.

Reply:
Prob `ssh_port` to go with simplicity and what the dev will prob think of first